forest/cli/src/cli/prelude.js

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019-2023 ChainSafe Systems
```

Comment:
How about adding a JS linter to the CI? I don't think most of us are proficient in JS, so at least such a tool would catch some bad practices.

Reply:
Good idea!